Assets/Scripts/InteractiveObject.cs

```diff
@@ -5,6 +5,7 @@
 [RequireComponent(typeof(AudioSource))]
 public class InteractiveObject : MonoBehaviour, IInteractive
 {
+    [Tooltip("Text that will display in the UI when the player looks at this object in the world.")]
     [SerializeField]
     protected string displayText = nameof(InteractiveObject);
     
```

Comment:
There's an unnecessary blank line on line 27. Don't really need the empty line on 30 either, but that's more up to personal preference I think, so long as you always put a blank after try/catch and debug.